src/schema/associations.ts

```diff
@@ -85,6 +86,16 @@ const associationLookup = {
       sampling_frequency: columns.get('sampling_frequency'),
     }
   },
+  coordsystem: async (file: BIDSFile, options: { maxRows: number }): Promise<Channels> => {
```

Comment:
This return type needs to match.


```suggestion
  coordsystem: async (file: BIDSFile, options: { maxRows: number }): Promise<{path: string, keys: string[]}> => {
```

Reply:
heh. that's embarassing. can you tell I don't usually write TypeScript?